src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,90 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
```

Comment:
Don't begin class or method implementation with empty line. Please remove the empty line at the start of the class definition.